releases/nightly-build/compose-files/source/docker-compose-nexus-base.yml

```diff
@@ -216,13 +216,12 @@ services:
       - edgex-network
     environment:
       # KUIPER_DEBUG: "true"
-      KUIPER_CONSOLE_LOG: "true"
-      KUIPER_REST_PORT: 48075
-      EDGEX_SERVICE_SERVER: http://edgex-core-data:48080
+      KUIPER__BASIC__CONSOLELOG: "true"
+      KUIPER__BASIC__RESTPORT: 48075
       EDGEX__DEFAULT__SERVER: edgex-app-service-configurable-rules
+      EDGEX__DEFAULT__SERVICESERVER: http://edgex-core-data:48080
       EDGEX__DEFAULT__TOPIC: events
       EDGEX__DEFAULT__PROTOCOL: tcp
       EDGEX__DEFAULT__PORT: 5566
-      EDGEX__DEFAULT__TYPE: zero
```

Comment:
Why removing this?

Reply:
The default setting is `zero`, so I deleted this setting